src/main/scala/viper/gobra/frontend/Desugar.scala

```diff
@@ -533,6 +551,23 @@ object Desugar {
       // translate pre- and postconditions
       val pres = decl.spec.pres map preconditionD(ctx)
       val posts = decl.spec.posts map postconditionD(ctx)
+      
+         val terminationMeasure= decl.spec.terminationMeasure match {
+        case Some(measure) => measure match {
+          case PTupleTerminationMeasure(vector) => Some(vector map terminationMeasureD(ctx))
+          case PUnderscoreCharacter()=> {
+            val src: Meta = meta(PUnderscoreCharacter())
+            Some(Vector( in.UnderscoreTerminationMeasure()(src)))
+          }
```

Comment:
The `meta` method obtains position information of a Node in the original text. This is useful to map the errors produced by viper to the corresponding Gobra code. For that reason, here you should not do `meta(PUnderscoreCharacter())`, as that tries to obtain information from a newly created node and, if I am not mistaken, fails beause the node is not in the original AST. Instead, you could do the following:
```scala
          case n: PUnderscoreCharacter => {
            val src: Meta = meta(n)
            Some(Vector( in.UnderscoreTerminationMeasure()(src)))
          }
```